justfile

```diff
@@ -84,3 +84,10 @@ checklinks *args:
                         -or -wholename 'site/overrides/main.html' \
                         -or -wholename 'site/blog/Ibis-version-3.1.0-release/index.html' \) \))
     lychee "${files[@]}" --base site --require-https {{ args }}
+
+# view the changelog for upcoming release (use --pretty to format with glow)
+view-changelog flags="":
```

Comment:
How about `just changelog`? Not a big deal either way.

Reply:
I wanted something that made it clear this was just for viewing the changelog, not writing it anywhere.